pkg/source/oss/inspection-type.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package oss
+
+import (
+	"math"
+
+	"github.com/GoogleCloudPlatform/khi/pkg/inspection"
+	"github.com/GoogleCloudPlatform/khi/pkg/source/oss/constant"
+)
+
+var OSSKubernetesLogFilesInspectionType = inspection.InspectionType{
+	Id:          constant.OSSInspectionTypeID,
+	Name:        "OSS Kubernetes Log Files",
+	Description: "Parsers for logs uploaded as the raw log files.",
+	Icon:        "assets/icons/anthos.png", // TODO: change! place holder
+	Priority:    math.MaxInt - 1000,
```

Comment:
Why -1000? This is a magic number so I'd suggest set a const value like const INF = math.MaxInt - 1000.

Reply:
This value is used for sorting inspection types on the dialog below.

![image](https://github.com/user-attachments/assets/f5534750-a5d7-4bfa-acd1-69a45466c362)

`math.MaxInt -1000` itself is not meaningful, but I wanted to make sure it's placed under all of the inspection types.

GKE currently has `math.MaxInt` as its priority, Cloud Composer has `math.MaxInt - 1`. But this way of specifying the order is not great. We may need to consider the better way.

Reply:
I don't have specific preference here but we may want to define constants in a single file?

Reply:
This problems are seen in multiple points of KHI project.
https://github.com/GoogleCloudPlatform/khi/blob/d5942963049b502cef742ed31d04a9d7736f2e36/pkg/source/gcp/task/form.go#L83C87-L83C126

Sorting forms are bit more terrible. I would address this unreasonable constant number issue in many sorts in KHI in the later refactoring soon.